server/util/role/role.go

```diff
@@ -14,13 +14,23 @@ const (
 	// None means that the user is not granted any role-based permissions for
 	// a particular group.
 	None Role = 0
+
 	// Developer means a user cannot perform certain privileged actions such
 	// as creating API keys and viewing usage data, but can perform most other
 	// common actions such as viewing invocation history.
+	//
+	// Developers have CAS write permissions and readonly AC permissions.
 	Developer Role = 1 << 0
+
 	// Admin means a user has unrestricted access within a group.
 	Admin Role = 1 << 1
 
+	// Writer is like Developer but also includes AC write perms.
+	Writer Role = 1 << 2
+
+	// Reader is like Developer but removes any cache write perms.
```

Comment:
Could this be more specific? 

// Reader allows CAS read and AC read but no writes.

or something similar?